Extension/package.json

```diff
@@ -362,7 +362,13 @@
             "clang-x64",
             "msvc-x86",
             "gcc-x86",
-            "clang-x86"
+            "clang-x86",
```

Comment:
Where is this ordering from? Would it be easier for users to choose from the list if it were sorted alphabetically?

Reply:
I copied the ordering from the c_cpp_properties.json. I can change the ordering there too to make it alphabetical.